README.md

```diff
@@ -86,6 +86,67 @@ wt cron history mongogron
 wt cron ls
 ```
 
+# Auth0 CLI: extend Auth0 with custom code (BETA)
+
+Auth0 hooks enable you to extend Auth0 platform with custom code. Auth0 CLI allows you to manage Auth0 hooks for your Auth0 account. 
+
+Auth0 hooks are webtasks associated with specific extensibility points of the Auth0 platform that are invoked by Auth0 runtime to execute your custom logic. 
+
+## Setup
+
+Follow the instructions from the Account > Webtasks panel on the Auth0 management dashboard to install and configure `wt-cli`. The package now includes `auth0` binary in addition to `wt`. 
+
+**NOTE** While Auth0 CLI is in BETA, the `wt-cli` tool must be installed from a branch of this repository instead of the public npm registry. Use `npm i -g auth0/wt-cli#auth0` to install the tool. The rest of the instructions on the Auth0 management portal applies without changes. 
+
+## Overview
+
+Auth0 CLI allows you to create, list, enable/disable, edit, and remove Auth0 hooks associated with specific extensibility points within the Auth0 platform, as well as receive real-time logging information generated by custom code. 
+
+The list of supported extensibility points will be growing over time. Currently the following extensibility points are supported: 
+
+[client-credentials-exchange](https://github.com/auth0/auth0-ext-compilers/blob/master/client-credentials-exchange.md)  
+[password-exchange](https://github.com/auth0/auth0-ext-compilers/blob/master/password-exchange.md)  
+[pre-user-registration](https://github.com/auth0/auth0-ext-compilers/blob/master/pre-user-registration.md)  
+[post-user-registration](https://github.com/auth0/auth0-ext-compilers/blob/master/post-user-registration.md)  
+
+For each of the extensibility points, there can be several Auth0 hooks created. A hook can be enabled or disabled, but only up to one hook per extensibility point can be enabled at a time. Disabled hooks are useful for staging new functionality. 
+
+## Synopsis
```

Comment:
I was on the fence about the approach. In retrospect I like your approach better for consistency sake with other commands. I will update.